plugins/search/src/components/SidebarSearchModal/SidebarSearchModal.tsx

```diff
@@ -16,17 +16,23 @@
 import React from 'react';
 import SearchIcon from '@material-ui/icons/Search';
 import { SidebarItem } from '@backstage/core-components';
+import { IconComponent } from '@backstage/core-plugin-api';
 import { SearchModal } from '../SearchModal';
 import { useSearch } from '../SearchContext';
 
-export const SidebarSearchModal = () => {
+export type SidebarSearchModalProps = {
```

Comment:
This needs exporting the whole way out too I think. So maybe `SidebarSearchModal/index.tsx` it will need the export for the prop type there too.

Reply:
exported, please check again.